pkg/config/setting.go

```diff
@@ -296,6 +297,53 @@ func ParseSetting(secrets, volCtx map[string]string, options []string, usePod bo
 	return &jfsSetting, nil
 }
 
+func (s JfsSetting) parseFormatOptions() ([][]string, error) {
+	options := strings.Split(s.FormatOptions, ",")
+	parsedOptions := make([][]string, 0)
+	for _, option := range options {
+		pair := strings.Split(strings.TrimSpace(option), "=")
```

Comment:
Some options are not "key=value" form, do not have value.

Reply:
Single-key options are supported.